src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchService.java

```diff
@@ -118,6 +119,12 @@ public VeilederPortefoljeStatusTall hentStatusTallForVeileder(String veilederId,
     }
 
     public Statustall hentStatustallForVeilederPortefolje(String veilederId, String enhetId, BrukerinnsynTilgangFilterType brukerinnsynTilgangFilterType) {
+        BrukerinnsynTilganger brukerInnsynTilganger = authService.hentVeilederBrukerInnsynTilganger();
+
+        if (brukerInnsynTilganger.harAlle() && BrukerinnsynTilgangFilterType.BRUKERE_SOM_VEILEDER_IKKE_HAR_INNSYNSRETT_PÅ == brukerinnsynTilgangFilterType) {
```

Comment:
Seems like `hentStatustallForVeilederPortefolje` is never called with `BRUKERE_SOM_VEILEDER_IKKE_HAR_INNSYNSRETT_PÅ` so I am not sure why we need this condition.